gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/Graph.java

```diff
@@ -61,13 +61,13 @@
     /**
      * Configuration key used by {@link GraphFactory}} to determine which graph to instantiate.
      */
-    public static final String GRAPH = "gremlin.graph";
+    String GRAPH = "gremlin.graph";
 
     /**
      * This should only be used by providers to create keys, labels, etc. in a namespace safe from users.
      * Users are not allowed to generate property keys, step labels, etc. that are key'd "hidden".
      */
-    public static class Hidden {
+    class Hidden {
```

Comment:
I did not restrict the scope. I can revert the changes but all those publics and statics were redundant, since they are inside a public interface. 

I also added a new GraphFeature for OrderabilitySemantics.

Reply:
ok makes sense.